data-structures-stacks/get-2nd-from-top.js

```diff
@@ -0,0 +1,26 @@
+/* exported get2ndFromTop */
+
+function get2ndFromTop(stack) {
+  let top;
+  let second;
+  // for if the stack is empty, return undefined.
+  if (stack.peek() === undefined) {
+    return undefined;
+  }
+  // Take most top, store it into top.
+  if (stack.peek() !== undefined) {
```

Comment:
Instead of checking the opposite of what you already checked use an `else` statment